os.h

```diff
@@ -0,0 +1,16 @@
+// Legacy BASIC
+// Utility types and functions.
+// Copyright (c) 2021-2 Nigel Perks
```

Comment:
Is that copyright OK? I'm happy if you want to put "contributed by ...." or similar.

Reply:
Yes, that copyright is fine. :)